src/NetworkPort.php

```diff
@@ -1163,8 +1163,12 @@ protected function showPort(array $port, $dprefs, $so, $canedit, $agg, $rand, $w
                             $relations_id = 0;
                             $oppositePort = NetworkPort_NetworkPort::getOpposite($netport, $relations_id);
 
-                            if ($oppositePort !== false) {
-                                $device2 = $oppositePort->getItem();
+                            if ($oppositePort === false) {
+                                break;
+                            }
+
+                            $device2 = $oppositePort->getItem();
+                            if ($device2 !== false) {
```

Comment:
Condition be negated with an break to avoid a big nested if here (like you did above).

Reply:
Yes, but I try to not change too many lines of code to limit the probability of conflicts during the `10.0/bugfixes`>`main` merge operation.